app/script/view_model/content/InputBarViewModel.js

```diff
@@ -524,6 +529,7 @@ z.viewModel.content.InputBarViewModel = class InputBarViewModel {
   }
 
   sendMessageEdit(messageText, messageEntity) {
+    const mentionEntities = [...this.currentMentions];
     this.cancelMessageEditing();
```

Comment:
We need to make a copy here, because `this.cancelMessageEditing()` will delete the array.